xmodule/capa_block.py

```diff
@@ -181,6 +192,21 @@ class ProblemBlock(
                "If the value is not set, infinite attempts are allowed."),
         values={"min": 0}, scope=Scope.settings
     )
+    grading_strategy = String(
+        display_name=_("Grading Strategy"),
+        help=_(
+            "Define the grading strategy for this problem. By default the last score "
+            "made by the student is taken."
+        ),
```

Comment:
Didn't we update this description? I forgot where I left the suggestion but it was something like:
```suggestion
        help=_(
            "Define the grading strategy for this problem. By default, it's the last submissions' score "
            "made by the student"
        ),
```

Reply:
It had been updated in `frontend-lib-content-components`, I already updated it here too